devices/Mhz19b/README.md

```diff
@@ -0,0 +1,57 @@
+﻿# MH-Z19B CO2-Sensor
+
+Binding for the MH-Z19B NDIR infrared gas module. The gas module measures the CO2 gas concentration in the ambient air.
+
+## Documentation
+
+[MH-Z19b Datasheet](https://www.winsen-sensor.com/d/files/infrared-gas-sensor/mh-z19b-co2-ver1_0.pdf)
+
+## Usage
+
```

Comment:
Missing here the usual  mention to add ref to Hardware.Esp32 and the code lines to configure the GPIOs for UART.

Reply:
See line #11